integration/unplugin/src/index.ts

```diff
@@ -252,6 +255,19 @@ const civetUnplugin = createUnplugin((options: PluginOptions = {}) => {
 
         return null;
       },
+      resolveId(id, importer) {
+        if (/\0/.test(id)) return null;
+        if (!isCivet(id)) return null;
+
+        const absolutePath = path.isAbsolute(id)
+          ? path.join(path.join(process.cwd(), viteConfig.root ?? ''), id)
+          : path.join(path.dirname(importer ?? ''), id);
```

Comment:
I'm pretty sure the latter case should use `path.resolve`, not `path.join`.

I pushed a new commit that cleans up this code. Could you check it?

Reply:
The result should be the same with `path.resolve` and `path.join` as we're already handling the case where `id` is an absolute path which is the only case where resolve and join differ

Reply:
Rest of the changes look good 👍